src/layout/Checkboxes/config.ts

```diff
@@ -25,7 +46,8 @@ export const Config = new CG.component({
         .setTitle('Alert on change')
         .setDescription('Boolean value indicating if the component should alert on uncheck'),
     ),
-  );
+  )
+  .addProperty(new CG.prop('summaryProps', CHECKBOX_SUMMARY_PROPS).onlyIn(Variant.Internal));
```

Comment:
See your point, need a little discussion to find a good solution :)

Reply:
Yep - these were my initial thoughts, I guess. :sweat_smile: I guess in hindsight (when reading the rest of the code) I think these could just be exported directly - but it doesn't make sense adding them as an internal property, as they never end up in `node.item` (which our types now say they do). It makes sense to only use these as an external type in the Summary2 component, but define them here in each separate component (for which their config belongs).

I've also mentioned earlier that I'd love to see some validation implemented for this (see #648), so that DevTools spits out errors/warnings if you configure overrides that are not possible for the target component.